examples/build.gradle

```diff
@@ -29,7 +29,7 @@ dependencies {
             project(':grpc-netty'),
             project(':grpc-protobuf'),
             project(':grpc-stub'),
-            'org.glassfish:javax.json:1.0.4'
+            'com.google.protobuf:protobuf-java-util:3.0.0-beta-2'
```

Comment:
Let's put this dep in the 'libraries' map of the root build.gradle and reference it as something like `libraries.protobuf_util` here.